mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -145,7 +146,7 @@
     private Snackbar mLastSnackbar;
     private Connection mConnection;
 
-    private Uri mPendingNfcData;
+    private Intent mPendingNfcIntent;
```

Comment:
When passing only the URI to BackgroundTasksManager, this change and everything related to it is superfluous.